### DISCUSSION TITLE
can't use `type: "bytes"` in ReadableStream

### DISCUSSION THREAD

**Discussion by jimmywarting** [DefinitelyTyped/DefinitelyTyped#54916] [Issues with a @types package]
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a245f923a8f79242a376b468350a677f43ed2c00/types/akamai-edgeworkers/index.d.ts#L676

should be `type?: string;`?

**Comment by thejohnfreeman** [comment]
Can you please edit the title of this issue to indicate that it is related to the `akamai-edgeworkers` package?
